### DISCUSSION TITLE
set expired date  of users in one port

### DISCUSSION THREAD

**Discussion by mohammadRezaeian** [FranzKafkaYu/x-ui#271] [Ideas]
When we create many users in one port 
It Isn't possible to set expired date for every user in one port.
Please add this option
Thank you.

**Comment by hossein-mohseni** [comment]
duplicated!

 #264

**Comment by FranzKafkaYu** [comment]
why should I develop this feature while you guys can make money from my free product,go away don't require more from me.

**Reply by hossein-mohseni** [reply]
How about if I pay you with tether?

**Reply by abaqqal** [reply]
> why should I develop this feature while you guys can make money from my free product,go away don't require more from me.

To support our servers and keep them alive, we need these features.
In Iran, we have a lot of problems such as transfer money to other counters to buy VPS. Pay Bribe to Internet police. And ....
For it case, we need to make money little.
Help us, please.
Also, I have another question.
Will you support this system in the future?

**Reply by hossein-mohseni** [reply]
> How about if I pay you with tether?

and i will help you to DeVos telegram bot for this project with fully support all the x-ui panel functionality

**Reply by FranzKafkaYu** [reply]
> > How about if I pay you with tether?
> 
> and i will help you to DeVos telegram bot for this project with fully support all the x-ui panel functionality

I can do this by myself.No need to bother you.

**Reply by hossein-mohseni** [reply]
> > > How about if I pay you with tether?
> > 
> > and i will help you to DeVos telegram bot for this project with fully support all the x-ui panel functionality
> 
> I can do this by myself.No need to bother you.

I konw 

Just I will be happy to help you and it will be my pleasure to work with you and to tell you your not alone all people in Iran are really Appreciate what you done🌹

**Reply by FranzKafkaYu** [reply]
> > > > How about if I pay you with tether?
> > > 
> > > 
> > > and i will help you to DeVos telegram bot for this project with fully support all the x-ui panel functionality
> > 
> > 
> > I can do this by myself.No need to bother you.
> 
> I konw
> 
> Just I will be happy to help you and it will be my pleasure to work with you and to tell you your not alone all people in Iran are really Appreciate what you done🌹

Thanks for your kind consideration and I am happy to hear such words from you.But I developed this just for personal use and I don't want to turn it to commercial.I know many of you use this for making money.And if we have something to do with money,we have pressure for pleasing Customers. Currently I don't have much time to handle this.

**Reply by hossein-mohseni** [reply]
Can I and my team help you in anything ?

I'm python and C# programmer and network manager and cyber security specialist.

at your service 🫡😄

**Reply by hossein-mohseni** [reply]
> > > > > How about if I pay you with tether?
> > > > 
> > > > 
> > > > and i will help you to DeVos telegram bot for this project with fully support all the x-ui panel functionality
> > > 
> > > 
> > > I can do this by myself.No need to bother you.
> > 
> > I konw
> > 
> > Just I will be happy to help you and it will be my pleasure to work with you and to tell you your not alone all people in Iran are really Appreciate what you done🌹
> 
> Thanks for your kind consideration and I am happy to hear such words from you.But I developed this is just for personal use and I don't want to turn it to commercial.I know many of you use this for making money.And if we have something to do with money,we have pressure for pleasing Customers. Currently I don't have much time to handle this.

I can edit the html css Javascript part of the code but unfortunately I can't edit the golang part and I can learn some so just to help you a little bit 😅

**Reply by Jabi-james** [reply]
> why should I develop this feature while you guys can make money from my free product,go away don't require more from me.

Exactly ..👍👍

**Comment by FranzKafkaYu** [comment]
latest release supports this feature.
